BrainPortal/config/environments/production.rb

```diff
@@ -17,7 +17,7 @@
 # GNU General Public License for more details.
 #
 # You should have received a copy of the GNU General Public License
-# along with this program.  If not, see <http://www.gnu.org/licenses/>.  
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
```

Comment:
do not change the default environments... these are to be adjusted by people who deploy CBRAIN